app/Http/Requests/backend/CreateBookRequest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Http\Requests\backend;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class CreateBookRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'title'       => 'required|min:6',
+            'category'    => 'required',
```

Comment:
1. `category ` em nên check thêm có tồn tại trong bảng `category` nữa (https://laravel.com/docs/5.5/validation#rule-exists)
2. `unit ` nên check thêm in, chỉ cho nhận một vài giá trị thôi và các giá trị mình sẽ define trong model (https://laravel.com/docs/5.5/validation#rule-in)
3. `year` thì mình validate theo kiểu: `date_format:"Y"`, không cần `min`, `max` đâu em, và trường `year` có càn required không?
4. `picture` mình có cần check size của image không?(https://laravel.com/docs/5.5/validation#rule-dimensions), và check trong document xem `picture` có cần `required` không